plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -294,11 +294,13 @@ export function getSourceFromLocation(
 export function wrapFields(
   type: Maybe<graphqlTypes.GraphQLObjectType & OtelPatched>,
   tracer: api.Tracer,
-  getConfig: () => GraphQLInstrumentationParsedConfig
+  getConfig: () => GraphQLInstrumentationParsedConfig,
+  ignoreMap: () => Map<string, boolean>
 ): void {
   if (
     !type ||
     typeof type.getFields !== 'function' ||
+    ignoreMap().has(type.name) ||
```

Comment:
Is the indirection required?

Reply:
OK, saw https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1134#discussion_r963077861 now.
The whole thing about config changing is such an edge case I think, but alright - I guess that's another topic.

It shouldn't be an issue if you never overwrite the variable. Map has a `clear()` method just for that.